common-lib/common/panels.libsonnet

```diff
@@ -0,0 +1,14 @@
+local g = import 'g.libsonnet';
```

Comment:
Not strictly necessary, but the more explicit `import './g.libsonnet'` will ensure that the correct file is always used, and gives the user a hint that this is a local file in this library.

```suggestion
local g = import './g.libsonnet';
```

Reply:
yes, thanks!